client/src/App.js

```diff
@@ -1,13 +1,30 @@
+import { useEffect, useState } from 'react'
+import { Route, Switch } from 'react-router-dom'
 import styled from 'styled-components/macro'
 import PosterList from './components/PosterList'
-import popularList from './example_data/popular.json'
 
 export default function App() {
+  const [popularSeries, setPopularSeries] = useState([])
+
+  useEffect(() => {
+    fetch('/api/series/popular')
+      .then(res => res.json())
```

Comment:
You could extract this into something like `src/services/series.js#getPopular` or directly look into [React-Query](https://react-query.tanstack.com/) and use that.

Reply:
I will have a look into React-Query later.